public/pages/AnomalyCharts/components/AlertsFlyout/__tests__/__snapshots__/AlertsFlyout.test.tsx.snap

```diff
@@ -1,554 +1,15 @@
-                      >
-                        <p
-                          class="alerts_flyout_p"
-                        >
-                          View anomaly detector alerts on the anomaly history graph. See details of all alerts over time from the monitor.
-                        </p>
-                      </div>
-                    </div>
-                  </div>
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
-        <div
-          class="euiFlyoutFooter"
-        >
-          <div
-            class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--alignItemsCenter euiFlexGroup--justifyContentFlexEnd euiFlexGroup--directionRow euiFlexGroup--responsive"
-          >
-            <div
-              class="euiFlexItem"
-            >
-              <a
-                class="euiButton euiButton--primary"
-                data-test-subj="setUpAlerts"
-                href="https://opensearch.org/docs/monitoring-plugins/alerting"
-                rel="noopener noreferrer"
-                target="_blank"
-              >
-                <span
-                  class="euiButtonContent euiButton__content"
-                >
-                  <span
-                    class="euiButton__text"
-                  >
-                    Explore Alerting 
-                    <svg
-                      aria-hidden="true"
-                      class="euiIcon euiIcon--small euiIcon-isLoading"
-                      focusable="false"
-                      height="16"
-                      role="img"
-                      viewBox="0 0 16 16"
-                      width="16"
-                      xmlns="http://www.w3.org/2000/svg"
-                    />
-                  </span>
-                </span>
-              </a>
-            </div>
-            <div
-              class="euiFlexItem"
-            >
-              <a
-                class="euiButton euiButton--primary euiButton--fill"
-                detectorid="GILVg3EBhaIh92zvX8uE"
-                detectorinterval="1"
-                detectorname="test detector"
-                href="/899c56c6-99d0-539b-8"
-                monitor="[object Object]"
-                rel="noreferrer"
-                unit="Minutes"
-              >
-                <span
-                  class="euiButtonContent euiButton__content"
-                >
-                  <span
-                    class="euiButton__text"
-                  >
-                    Edit alert settings
-                  </span>
-                </span>
-              </a>
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-    <div
-      aria-hidden="true"
-      data-aria-hidden="true"
-      data-focus-guard="true"
-      style="width: 1px; height: 0px; padding: 0px; overflow: hidden; position: fixed; top: 1px; left: 1px;"
-      tabindex="0"
-    />
-  </div>
-</div>
+<div
+  aria-hidden="true"
```

Comment:
Seems we changed a lot for alerts flyout. Looks quite simple now. Can you confirm if the `AlertsFlyout` snap is correct or not?

Reply:
Investigating this now, ConfirmModal snap also had a lot removed. Need to see why testing isn't opening flyout/modal.

Reply:
Still no clear answer on this. I opened an issue: https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/issues/231 and will continue investigating with the help on others on dashboard team. For the failing integ-test, the correct UI is still there but for some reason the aria-label for "Daily max" isn't present in the element. I will make a change to fix it on functional-test-repo after this is merged

Reply:
Similar thing happened in ISM 2.0 update with their `CreateState` snap https://github.com/opensearch-project/index-management-dashboards-plugin/commit/7352cdbc9372dab2062538d5f292884f4a10e5a9#diff-7fde899dd6c89f4ee9fda19ed4cd3eff64cb1e0ba4e6e915d1e6631b42201351R4